CMakeLists.txt

```diff
@@ -27,11 +30,49 @@ include_directories(rapidxml)
 include_directories(${CMAKE_INCLUDE_PATH})
 link_directories(${CMAKE_LIBRARY_PATH})
 
-find_package(LAMMPS REQUIRED)
-include_directories(${LAMMPS_INCLUDE_DIRS})
+#find_package(LAMMPS REQUIRED)
+#include_directories(${LAMMPS_INCLUDE_DIRS})
+
+set(LAMMPS_LIBRARIES "${LAMMPS_LIBRARY_DIR}/liblammps.so")
+# Create an imported target (if necessary)
+add_library(LAMMPS::Library SHARED IMPORTED)
+set_target_properties(LAMMPS::Library PROPERTIES
+  IMPORTED_LOCATION "${LAMMPS_LIBRARIES}"
+  INTERFACE_INCLUDE_DIRECTORIES "${LAMMPS_INCLUDE_DIR}"
+)
+
+#ww # Specify the static library file
+#ww set(LAMMPS_LIBRARIES "${LAMMPS_LIBRARY_DIR}/liblammps.a")
+#ww 
+#ww # Create an imported target
+#ww add_library(LAMMPS::Library STATIC IMPORTED)
+#ww set_target_properties(LAMMPS::Library PROPERTIES
+#ww   IMPORTED_LOCATION "${LAMMPS_LIBRARIES}"
+#ww   INTERFACE_INCLUDE_DIRECTORIES "${LAMMPS_INCLUDE_DIR}"
+#ww )
+
+#ww # Specify the static library file
+#ww set(LAMMPS_LIBRARIES "${LAMMPS_LIBRARY_DIR}/liblammps.a")
+#ww 
+#ww # Create an imported target
+#ww add_library(LAMMPS::Library STATIC IMPORTED)
+#ww set_target_properties(LAMMPS::Library PROPERTIES
+#ww   IMPORTED_LOCATION "${LAMMPS_LIBRARIES}"
+#ww   INTERFACE_INCLUDE_DIRECTORIES "${LAMMPS_INCLUDE_DIR}"
+#ww )
+
+message( STATUS "*** !!!!!! PREFIX            : " ${PREFIX} )
+message( STATUS "*** !!!!!! CMAKE_INCLUDE_PATH: " ${CMAKE_INCLUDE_PATH})
+message( STATUS "*** !!!!!! LAMMPS_INCLUDE_DIR: " ${LAMMPS_INCLUDE_DIR})
+message( STATUS "*** !!!!!! CMAKE_LIBRARY_PATH: " ${CMAKE_LIBRARY_PATH})
+message( STATUS "*** !!!!!! LAMMPS_LIBRARY_DIR: " ${LAMMPS_LIBRARY_DIR})
+message( STATUS "*** !!!!!! LAMMPS_LIBRARIES: " ${LAMMPS_LIBRARIES})
+
 
 # include VERBOSE for debugging output
-add_definitions(-std=c++11 -O3 -DLAMMPS_EXCEPTIONS -DLAMMPS_LIB_MPI)# -DVERBOSE)
+add_definitions(-std=c++11 -O3 -DLAMMPS_EXCEPTIONS -DLAMMPS_LIB_MPI) -DLAMMPS_MEMALIGN=64# -DVERBOSE)
```

Comment:
Typo on closing bracket ? 
`add_definitions(-std=c++11 -O3 -DLAMMPS_EXCEPTIONS -DLAMMPS_LIB_MPI -DLAMMPS_MEMALIGN=64)`

Reply:
Thanks @arn-all it looks like it...